Website/src/rendering/src/components/Sponsors/SponsorsGrid.tsx

```diff
@@ -10,8 +16,15 @@ type SponsorsGridProps = ComponentProps & {
 };
 
 const SponsorsGrid = (props: SponsorsGridProps): JSX.Element => {
+  const { sitecoreContext } = useSitecoreContext<SitecoreContextValue>();
+
+  const isPageEditing = sitecoreContext.pageState === LayoutServicePageState.Edit;
+  const hasSponsors = props.fields;
```

Comment:
`hasSponsors` is an object with that code. We could use `!!props.fields` for it to be a boolean and consume less memory.